.github/ISSUE_TEMPLATE/release.md

```diff
@@ -42,7 +42,7 @@ $ go build -tags tti ./cmd/tti-lw-cli
 - [ ] Then export the CLI documentation using the following command:
 
 ```bash
-$ HOME='$HOME' ./tti-lw-cli gen-yaml-doc -o /path/to/lorawan-stack-docs/doc/data/
+$ HOME='$HOME' ./tti-lw-cli gen-yaml-doc -o ../lorawan-stack-docs/doc/data/
```

Comment:
There's no reason to assume that `lorawan-stack-docs` is located in the parent folder here. Leave as is.

```suggestion
$ HOME='$HOME' ./tti-lw-cli gen-yaml-doc -o /path/to/lorawan-stack-docs/doc/data/
```

Reply:
@benolayinka then why do we assume it in the rest of the template then? 

> - [ ] Copy generated files to `lorawan-stack-docs` with the following command:
> ```bash
> $ cp api/ttn.lorawan.v3/*.yml ../lorawan-stack-docs/doc/data/api/ttn.lorawan.v3/
> $ cp api/tti.lorawan.v3/*.yml ../lorawan-stack-docs/doc/data/api/tti.lorawan.v3/
> ```

We should be consistent with one or the other

Reply:
that's my fault. should all be /path/to